tools/packaging/common/envoy_bootstrap.json

```diff
@@ -627,6 +627,29 @@
       }
     }
   }
+  {{- else if .openCensusAgent }}
+  ,
+  "tracing": {
+    "http": {
+      "name": "envoy.tracers.opencensus",
+      "typed_config": {
+        "@type": "type.googleapis.com/envoy.config.trace.v3.OpenCensusConfig",
+        "ocagent_exporter_enabled": true,
+        "ocagent_address": "{{ .openCensusAgent }}",
+        "incoming_trace_context": {{ .openCensusAgentContexts }},
+        "outgoing_trace_context": {{ .openCensusAgentContexts }},
+        "trace_config": {
+          "constant_sampler": {
+            "decision": "ALWAYS_PARENT"
+          },
+          "max_number_of_annotations": 200,
```

Comment:
should this be configurable?

Reply:
@nrjpoddar and I discussed in the API PR and I ended up removing the tuning vars from the API. If you think it is worthwhile, I can add env variable tuning of these, but it doesn't appear to have a real use-case by users.

https://github.com/istio/api/pull/1591